.travis.yml

```diff
@@ -12,3 +12,9 @@ matrix:
       install:
         pip install -r $TRAVIS_BUILD_DIR/requirements-testing.txt
       script: tox -c $TRAVIS_BUILD_DIR
+    - language: python
+      python:
+        - 2.7
+      install:
+        pip install -r $TRAVIS_BUILD_DIR/requirements-testing.txt
```

Comment:
Shouldn't this actually be `$TRAVIS_BUILD_DIR/requirements.txt`? The sphinx build doesn't need `requirements-testing.txt`.